builds/misc/images.yaml

```diff
@@ -190,177 +195,6 @@ jobs:
       archName: arm64
       targetArch: aarch64
 
-################################################################################
-  - job: windows
-################################################################################
-    displayName: Windows
-    timeoutInMinutes: 90
-    pool:
-      name: Azure-IoT-Edge-Core
-      demands:
-        - Build-Image -equals true
-        - win-rs5
-    workspace:
-      clean: all
-    steps:
-      - template: ../templates/install-dotnet2.yaml
-      - template: ../templates/install-dotnet3.yaml
-
-      - script: echo $(registry.password)|docker login "edgebuilds.azurecr.io" -u "$(registry.user)" --password-stdin
-        displayName: Docker Login
-
-      - powershell: scripts/windows/build/Publish-Branch.ps1 -Configuration:"$(Build.Configuration)" -PublishTests:$False -UpdateVersion
```

Comment:
I will delete all the yaml and ps1 windows files in another PR. This will mean we also need to disable the checkin / CI for tests on windows. 

I need to confirm, but think our E2E tests should probably stay because we will be running linux containers on windows. We will need to alter the test pipeline to run these linux containers.

Reply:
Why did we decide not to build the broker for windows? Is this part of a greater agenda to move away from Windows and towards linux containers on windows?

Reply:
Yep that is the reason.